documentation/documentation/docs/tutorial-basics/create-a-page.md

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
just get rid of all tutorials maybe

Reply:
https://github.com/SELab-2/UGent-3/pull/250/commits/c2af2565e6508f3e8caa588f821307480a79c078